downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,422 @@
+|--------------------------------------------------------|----------------------------------------------------------------|
+| `array.getType()`                                      | This will return an `IntersectionType` instead of `ArrayType`. |
+| `getDescribingType()` on `ReadonlyIntArray` type       | This will return an `IntersectionType` instead of `ArrayType`. |
+| `getImpliedType()` on received `IntersectionType` type | This will return an `ArrayType`.                               |      
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.8.0+label%3ATeam%2FjBallerina+label%3AType%2FBug+is%3Aclosed).
+
+## Standard library updates
+
+### New features
+
+#### `mqtt` package
+- Introduced the `mqtt` standard library package, which provides an implementation to interact with message brokers using the MQTT protocol.
+
+#### `java.jms` package
+- Introduced the `java.jms` standard library package, which provides an implementation to interact with message brokers using the JMS protocol.
+
+#### `graphql` package
+
+- Introduced the `DataLoader` functionality.
+
+### Deprecations
+
+#### `graphql` package
+
+- Deprecated the `executeWithType()` method of the `graphql:Client`.
+
+### Improvements
+
+#### `graphql` package
+
+- Added support for the `@deprecated` directive to output objects defined using record types.
+- Added support for printing the GraphiQL URL to `stdout`.
+- Added support for generating a subgraph SDL schema at compile time.
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%222201.8.0%22+label%3AType%2FBug).
+
+## Developer tools updates
+
+### New features
+
+#### Language Server
+
+- Add custom completions for HTTP services.
+
+#### CLI
+
+- Added a new `bal tool` command to manage tools that extend the functionality of the CLI. Tools can be pulled from the Ballerina Central and are managed using the `bal tool` command. For more information, see [Tool commands](https://ballerina.io/learn/cli-documentation/cli-commands/#tool-commands).
+
+#### OpenAPI tool
+- Added support to generate Ballerina client and service declarations from OpenAPI v3.1.x definitions.
+
+### Improvements
+
+#### Language Server
+
+- Suggest Ballerina Central packages when they are partially typed.
+- Improve sorting in the record type descriptor node context.
+- Introduce a code action to add local module dependencies to the `Ballerina.toml` file.
+- Introduce a code action to change the variable type of a `let` expression.
+- Introduce a code action to create a function for the expression of a `select` clause.
+- Improve completions in the service declaration node context.
+- Improved the LS simulator.
```

Comment:
Improve? I would actually change the rest to this form.

Reply:
Updated. The guideline is to use past tense in the release note to give the idea that it is already done in the release.